app/main/util/data_cleaning.py

```diff
@@ -0,0 +1,92 @@
+from ..dataModel.user_recommendation import UserRecommendation
+from ..dataModel.category import Category
+from ..dataModel.merchant_category import Merchant_Category
+from ..dataModel.merchant_product import Merchant_Product
+from ..dataModel.merchant import Merchant
+from ..dataModel.product import Product
+from ..dataModel.user_purchase import UserPurchase
+from ..datkaModel.user import User
+
+from ..database import DB
+
+import numpy as np
```

Comment:
Need to add numpy and pandas in `requirements.txt` file otherwise service does not run

Reply:
Ohh yeah forgot that too :P